ui/src/views/network/CreateIsolatedNetworkForm.vue

```diff
@@ -40,7 +40,7 @@
               <tooltip-label :title="$t('label.displaytext')" :tooltip="apiParams.displaytext.description"/>
             </template>
             <a-input
-             v-model:value="form.displaytext"
+             v-model:value="form.name"
```

Comment:
@rahulbcn27 , I don´t mean to take the name unconditionally, but only when the users decides to submit without a value added in the `displaytext` field.

I would recommend to make the field optional on the command and implement the check on the service.

Reply:
Thanks @DaanHoogland for the comment. I am implementing the change suggested by you and Rohit.